build.py

```diff
@@ -390,7 +390,7 @@ def main():
     else:
         datasets = split_dataset_days()
 
-    pickle.dump(datasets,open(os.path.join(DATASET_FOLDER,'datasets.dat'),'wb'))
+    pickle.dump(datasets, open(os.path.join(DATASET_FOLDER,'datasets.dat'),'wb'))
```

Comment:
I plan to run Black over the code base soon. That will sort out all this kind of stuff.